src/egon/data/airflow/dags/pipeline.py

```diff
@@ -672,6 +673,18 @@
         ]
     )
 
+    # import scenario status2019 from backup
+    import_status2019 = Import_Status2019(
+        dependencies=[
+            storage_etrago,
+            hts_etrago_table,
+            fill_etrago_generators,
+            household_electricity_demand_annual,
+            cts_demand_buildings,
+            emobility_mit,
+        ]
+    )
```

Comment:
Why do you need all these dependencies?

Reply:
These were the dependencies used for the low and medium flexibility scenarios. Since the creation concept of our new scenarios is the same, it makes sense to me to use the same dependencies. Now that you mentioned this, I think it is better to have this import status2019 and the task that creates the new scenarios in the same dataset.

Reply:
I meant these changes: 723295a51a454eb9faee84161fd0ae4d86c24017